addons/website_version/models/version.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+from openerp import fields, models, api
+from openerp.exceptions import Warning
+from openerp.http import request
+from openerp.tools.translate import _
+
+
+class version(models.Model):
+
+    _name = "website_version.version"
+    
+    name = fields.Char(string="Title", required=True)
+    view_ids = fields.One2many('ir.ui.view', 'version_id', string="view_ids", copy=True)
+    website_id = fields.Many2one('website', ondelete='cascade', string="Website")
+    create_date = fields.Datetime('Create Date')
+
+    _sql_constraints = [
+        ('name_uniq', 'unique(name, website_id)', _('You cannot have multiple versions with the same name in the same domain!')),
+    ]
+
+    @api.one
+    def unlink(self):
```

Comment:
Unlink works in batch. Setting it to one breaks the batch behavior. Moreover the return won't be correct ([True] insteas of True, if you want to test the result for example).

Here you can perform  your search using self.ids for example, instead of self.id .